src/js/controller/controller.js

```diff
@@ -730,6 +734,10 @@ Object.assign(Controller.prototype, {
 
         function _attachMedia() {
             // Called after instream ends
+            const adConfig = _model.get('advertising');
+            if (adConfig && adConfig.outstream) {
+                _programController.trigger(MEDIA_COMPLETE, {});
+            }
```

Comment:
For these case it probably makes sense to add these methods to the instance here and extend in https://github.com/jwplayer/jwplayer-commercial/blob/v8.5.2/src/js/controller/controller.js

Reply:
Extending `attachMedia` works, but there's no reference to `_programController`. Any idea how to trigger `MEDIA_COMPLETE` from commercial?